src/server/getters/get-profit-loss.ts

```diff
@@ -23,16 +23,17 @@ export interface PLBucket {
   profitLoss?: ProfitLoss | null;
 }
 
-export function calculateBucketProfitLoss(augur: Augur, trades: Array<TradingHistoryRow>, buckets: Array<PLBucket>): Array<PLBucket> {
+export type TradeRow = TradingHistoryRow & { type: string, maker: boolean};
```

Comment:
I'm not familiar with this syntax; what does the single `&` mean here?

Reply:
It's a union type:

It's TradingHistoryRow plus a type field, and a maker field.